content/en/examples/controllers/job-success-policy-example.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: batch/v1
+kind: Job
+spec:
+  parallelism: 10
+  completions: 10
+  completionMode: Indexed # Required for the feature
+  successPolicy:
+    rules:
+      - succeededIndexes: 0-2
+        succeededCount: 1
+  template:
+    spec:
+      containers:
+      - name: main
+        image: python
+        command:          # The jobs succeed as there is one succeeded index
+                          # among indexes 0, 1, and 2.
+          - python3
+          - -c
+          - |
+            import os, sys
+            if os.environ.get("JOB_COMPLETION_INDEX") == "1":
+              sys.exit(0)
+            else:
+              sys.exit(1)
```

_optional advice_

Here's a way to split the Pods in a plausible, fictional way: 
```suggestion
            if (int(os.environ.get("JOB_COMPLETION_INDEX")) <= 2:
              # simulate three controller Pods that always succeed
              # the overall Job succeeds once two controllers have exited with success
              sleep(30) # simulate worker supervision
              sys.exit(0) # “controllers“ are always successful
            else:
              # simulate partial success for “workers”
              import random
              sys.exit(random.randint(0, 1))
```

---

I changed the successPolicy example so that we can clarify the interval mechanism.